### DISCUSSION TITLE
Проблема кеширование файла экспорта ресурса

### DISCUSSION THREAD

**Discussion by shevl-dev** [moonshine-software/moonshine#1094] [Ideas]
Файл, скачиваемый по запросу на адрес вида **/admin/resource/seo-resource/handler/export-handler**, кешируется и при повторном экспорте скачивается тот же файл, кажется запрос даже до хендлера не доходит.

Предлагаю в URL кнопки экспорта добавлять какой-то формальный уникальный параметр, например **timestamp** с текущей датой и временем, или какой-то хеш.

**Comment by lee-to** [comment]
Дело в nginx у тебя видимо стоит что то вроде add_header Cache-Control "public, max-age=2629000, no-transform, immutable";
Можем добавлять хеш, подумаю, пока просто настрой кеширование, так как скорее всего nginx не даст переопределить этот заголовок тебе в коде при респонсе на скачивание

**Reply by shevl-dev** [reply]
Только у меня apache, самая стандартная конфигурация, которая предоставляется в Open Server, ничего не кастомизировал под себя. Думаю там в целом стандартный апачевский конфиг.
